tests/test_datasets/test_dataset.py

```diff
@@ -262,6 +262,19 @@ def test_get_data_corrupt_pickle(self):
         self.assertIsInstance(xy, pd.DataFrame)
         self.assertEqual(xy.shape, (150, 5))
 
+    def test_load_metadata(self):
+        _compare_dataset = openml.datasets.get_dataset(
+            2, download_data=False, download_features_meta_data=True, download_qualities=True
+        )
+
+        _dataset = openml.datasets.get_dataset(
+            2, download_data=False, download_features_meta_data=False, download_qualities=False
```

Comment:
Hm, interesting. So the data is already available on the local drive, so you would not download it again, so the argument name is wrong. This should actually be called `load_features_meta_data` then, I guess?

Reply:
Not sure what you mean exactly. Calling `get_dataset(download_features_meta_data=True)` will download features meta_data or load it from the cache. This works similarly for `download_data` and `download_qualities`. Technically, all of these are a bit misleading, but considering that it is a cache I would say it is okay. 

If you referring to the function call below, `OpenMLDataset.load_metadata(features=True)`, the code will always download the data. I think I should also add a cache for that as `load_data` also caches the data or loads from the cache if available. I will change this accordingly.

Reply:
I have added cache usage to `OpenMLDataset.load_metadata`. It is now in line with `get_dataset()` and `OpenMLDataset.get_data`'s `_load_data`.

Reply:
Nevermind, I'm not sure whether my comment makes too much sense.

Reply:
It was a good comment. I have changed a few things to adjust to your comment and I the code is now better/clearer.